EssentialFeed/EssentialFeedTests/Image Comments/API/ImageCommentsMapperTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsMapperTests: XCTestCase {
+	func test_map_throwsErrorOnNon200HTTPResponse() throws {
+		let json = makeItemsJSON([])
+		let samples = [199, 300, 400, 500]
+
+		try samples.forEach { code in
+			XCTAssertThrowsError(
+				try ImageCommentsMapper.map(json, from: HTTPURLResponse(statusCode: code))
+			)
+		}
+	}
+
+	func test_map_throwsErrorOn200HTTPResponseWithInvalidJSON() throws {
```

Comment:
Same here for the name, it should be `OnNon2xx` instead of `On200`
```suggestion
	func test_map_throwsErrorOnNon2xxHTTPResponseWithInvalidJSON() throws {
```

Reply:
Renamed.